src/test/utils/helpers.js

```diff
@@ -24,9 +26,11 @@ export const generateArticle = async ({ authorId }) => ({
   title: faker.lorem.word(),
   body: faker.lorem.sentences(),
   image: faker.image.imageUrl(),
-  publishedDate: new Date(),
+  publishedDate: '2019-07-23T11:48:21.224Z',
```

Comment:
I think you should return the publishedDate to new date, other tests might rely on this

Reply:
This change is a result of pair-programming with Princewill where we encountered a difference in the time format used by sequelize and chai assertions.

Returning the `new Date()` function will lead to not using this method to generate an article owing to the fact that an article should be created without a `published date` set on creation.

But again when considering an update operation, a user may want to update an article that is already published which will require that the article already has a published date.

So this change is to mimic an expected behaviour during testing and it was tested that fits in other tests that depend on it at the time of this implementation